scripts/audit-deps.js

```diff
@@ -61,14 +61,16 @@ if (auditReport) {
     }
   }
 
-  for (const advId of Object.keys(auditReport.advisories)) {
-    const adv = auditReport.advisories[advId];
-
-    if (exceptions.includes(adv.url)) {
-      ignoredIssues.push(adv);
-      continue;
+  if (auditReport.advisories) {
+    for (const advId of Object.keys(auditReport.advisories)) {
+      const adv = auditReport.advisories[advId];
+
+      if (exceptions.includes(adv.url)) {
+        ignoredIssues.push(adv);
+        continue;
+      }
+      blockingIssues.push(adv);
     }
-    blockingIssues.push(adv);
   }
```

Comment:
@eviljeff would you mind to remove this part from this PR?

as it is in this PR, it does basically just silence the script when running on npm v8, even if some sec advisory was actually listed in the output of the `npm audit --json` command.

You should be able to just remove it and the CI jobs should still pass, because we got rid of a sec advisory that was triggered by an unused dev dependency, and I guess that may have been the one likely triggering the unexpected failure in the CI jobs for this PR before.

In any case, I also filed #2470 to track the need to update the script to take the new format introduced in npm v8 into account and make sure the script will work in both npm v7 and npm v8, and also opened #2471 with some changes to fix that issue, and so we should be able to get that fixed in the master branch too soon.